packages/bot/src/scratch/blocks/Binary/Indicators/rsi_statement.js

```diff
@@ -2,6 +2,8 @@ import { expectValue } from '../../../shared';
 import { translate }   from '../../../../utils/lang/i18n';
 
 Blockly.Blocks.rsi_statement = {
+    protected_statements : ['STATEMENT'],
+    required_child_blocks: ['input_list', 'period'],
```

Comment:
@aaron-binary it would be better if you read these `required_child_blocks` from some constants somewhere, so instead of adding them in all blocks

Reply:
They're block-specific, so while some of the indicator statement blocks share the same `required_child_blocks` they may differ for others. I think for clarity it's easier to keep this defined on a per block basis rather than adding a dependency on other blocks (by inheriting from them).